docs/source/basic/intro.rst

```diff
@@ -0,0 +1,90 @@
+~~~~~~~~~~~~~
+
+A Abstract Interface
+   It describing parallel execution on multiple hierarchy levels. It allows to implement a mapping to various hardware architectures but is no optimal mapping itself.
+
+Sustainably
+   *alpaka* decoupled the application from the availability of different accelerator frameworks in different versions, such as OpenMP, CUDA, HIP, etc. (50% on the way to reach full performance portability).
+
+Heterogeneity
+   An identical algorithm / kernel can be executed on heterogeneous parallel systems by selecting the target device. This allows the best performance for each algorithm and/or a good utilization of the system without major code changes.
+
+Maintainability
+   *alpaka* allows to provide a single version of the algorithm / kernel that can be used by all back-ends. There is no need for "copy and paste" kernels with different API calls for different accelerators. All the accelerator dependent implementation details are hidden within the *alpaka* library.
+
+Testability
+   Due to the easy back-end switch, no special hardware is required for testing the kernels. Even if the simulation itself will always use the *CUDA* back-end, the tests can completely run on a CPU. As long as the *alpaka* library is thoroughly tested for compatibility between the acceleration back-ends, the user simulation code is guaranteed to generate identical results (ignoring rounding errors / non-determinism) and is portable without any changes.
+
+Optimizable
+   Everything in *alpaka* can be replaced by user code to optimize for special use-cases.
+
+Extensible
+   Every concept described by the *alpaka* abstraction can be implemented by users. Therefore it is possible to non-intrusively define new devices, queues, buffer types or even whole accelerator back-ends.
+
+Data Structure Agnostic
+   The user can use and define arbitrary data structures.
+
+alpaka is not ...
+~~~~~~~~~~~~~~~~~
+
+An automatically optimal mapping of algorithms / kernels to various acceleration platforms
+   Except in trivial examples an optimal execution always depends on suitable selected data structure. An adaptive selection of data structures is a separate topic that has to be implemented in a distinct library.
+
+Automatically optimizing concurrent data accesses
+   *alpaka* does not provide feature to create optimized memory layouts.
+
+Handling or hiding differences in arithmetic operations
+   For example, due to **different rounding** or different implementations of floating point operations, results can differ slightly between accelerators.
+
+Guaranteeing any determinism of results
+   Due to the freedom of the library to reorder or repartition the threads within the tasks it is not possible or even desired to preserve deterministic results. For example, the non-associativity of floating point operations give non-deterministic results within and across accelerators.
+
+The *alpaka* library is aimed at parallelization within nodes of a cluster.
```

Comment:
Maybe add explicitly smth like "... at parallelization on shared memory, i.e. within nodes of a cluster"